src/components/actionSheet/index.js

```diff
@@ -158,17 +182,19 @@ export default class ActionSheet extends BaseComponent {
     }
   }
 
-  renderSheet() {    
+  renderSheet() {
+    const {renderTitle} = this.props;
+    const {sheetStyle} = this.getThemeProps();
     return (
-      <View bg-white>
-        {this.renderTitle()}
+      <View style={[styles.sheet, sheetStyle]} >
+        {renderTitle ? renderTitle() : this.renderTitle()}
```

Comment:
You are not checking if renderTitle is a function before you call it?

Reply:
Done